js/up.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+const form = document.forms.userProfile;
+const ageInput = document.querySelector('#age').value;
+const heightInput = document.querySelector('#height').value;
+const weightInput = document.querySelector('#weight').value;
+const bmiInput = document.querySelector('#bmi').value;
+const cityInput = document.querySelector('#city').value;
+const stateInput = document.querySelector('#state').value;
+const fitnessGoalsInput = document.querySelector('#fitnessGoals').value;
+
+
+function populateLocalStorage() {
+    localStorage.setItem('age', 'ageInput');
```

Comment:
Abby, this won't work. The function will not recognize the variables outside of its scope. This is why I have the (.value) designations inside of my function.  Take a look at my js in for new_acct. If you have questions or need help let me know.